plugins/woo-ai/src/product-description/product-description-button-container.tsx

```diff
@@ -18,11 +18,19 @@ import { recordTracksFactory, getPostId } from '../utils';
 
 const DESCRIPTION_MAX_LENGTH = 300;
 
-const getApiError = () => {
-	return __(
-		`❗ We're currently experiencing high demand for our experimental feature. Please check back in shortly.`,
-		'woocommerce'
-	);
+const getApiError = ( error: string ) => {
+	switch ( error ) {
+		case 'connection_error':
+			return __(
+				'❗ We were unable to reach the experimental service. Please check back in shortly.',
```

Comment:
A slightly altered error if the issue was with creating the connection, although probably not actually necessary. Any opinions on what this should be, or if it should just be the same as the default?

Reply:
Honestly, I think I like this _new_ error message as the default:
`We were unable to reach the experimental service. Please check back in shortly.`
Unless I'm mistaken, we don't actually have a check for a specific error code that would hint at our AI services being at capacity? If that's the case, I personally like keeping it simple with everything returning "We were unable to reach the experimental service. Please check back in shortly."

Reply:
True, although we'll presumably have more control over this when we switch to our own endpoints. We could get more granular at that point.